qemu/tests/boot_with_max_mem.py

```diff
@@ -0,0 +1,32 @@
+from virttest import env_process, utils_misc
+
+
+def run(test, params, env):
+    """
+    Test steps:
+    1) Check whether the host memory is greater than boot guest memory:
+    If the host memory is greater than boot guest memory,
+    continue to test the case,Otherwise, skip the test case
+    2) Boot guest with max memory with two cpus
+    3) Check guest dmesg, no call trace message
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+    host_mem = utils_misc.get_usable_memory_size() // 1024 // 1024
+    boot_mem = params.get("boot_mem")
+    if host_mem < int(boot_mem[0:-1]):
+        test.cancel("The host memory is less than %s of boot guest memory" % boot_mem)
+    params["mem"] = params.get("boot_mem")
+    params["smp"] = int(params.get("boot_smp"))
+    params["vcpu_maxcpus"] = int(params.get("boot_smp"))
```

Comment:
No need, L21 ~L23

Reply:
@yanan-fu , I have removed L21 ~ L23, please check it, thanks